packages/catalog-model/examples/acme/infrastructure-group.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Group
+metadata:
+  name: infrastructure
+  description: The infra business unit
+spec:
+  type: business-unit
```

Comment:
both these levels (infrastructure and boxoffice/backstage) have the same type - i don't know enough about orgs to know if there's some other appropriate term for something between a team and a business unit :)

Reply:
Is this field freeform text? How about department?

Reply:
Yeah, or rather more like an enum - it is up to the org themselves to decide what terminology they use, we don't enforce one but lowercase-with-dashes is good in any way. Department sounds fine!